rotkehlchen/tests/conftest.py

```diff
@@ -33,6 +33,7 @@
 
 TESTS_ROOT_DIR = Path(__file__).parent
 SUBPROCESS_TIMEOUT = 30
+os.environ['POLARS_ALLOW_FORKING_THREAD'] = '1'  # ignore warning about fork and polars
```

Comment:
why did this pop out right now? Oh this is develop. Okay then this is another problem.